benches/oneshot/sort.rs

```diff
@@ -45,9 +45,9 @@ async fn main() -> Result<(), BoxError> {
 
     let start = Instant::now();
     let result = try_join_all(vec![
-        GenerateSortPermutation::new(&shares[0], num_bits).execute(ctx0),
-        GenerateSortPermutation::new(&shares[1], num_bits).execute(ctx1),
-        GenerateSortPermutation::new(&shares[2], num_bits).execute(ctx2),
+        generate_sort_permutation(ctx0, &shares[0], num_bits),
+        generate_sort_permutation(ctx1, &shares[1], num_bits),
+        generate_sort_permutation(ctx2, &shares[2], num_bits),
     ])
     .await?;
```

Comment:
```suggestion
    )
    .await;
```
With the above, that is.

Reply:
#265